.github/workflows/_build_rosetta.yaml

```diff
@@ -1,13 +1,38 @@
 name: ~build Rosetta container
 
 on:
+  workflow_dispatch:
```

Comment:
Just a reminder for myself to ensure workflow_dispatch is removed before merging.

Reply:
OOC, is there any reason not to allow workflow_dispatch on a reusable workflow? Just curious since if we "matrixify" the rosetta build/test for pax/t5x, someone could test the individual build/test workflows by dispatching the reusable _build_rosetta.yaml or _test_rosetta.yaml. But if someone needs to run the "matrixified" nightly-rosetta-build.yaml, we'd have to add logic to test one or the other OR make a concrete nightly_rosetta_t5x_build.yaml in addition to the "matrixified" nightly_rosetta_build.yaml, which would repeat a lot of the scaffolding

Reply:
This is meant to avoid duplication of the `inputs` section and to simplify input processing logic in the reusable workflow. I wish to keep the messy part in the 'frontpage' workflows while extracting the clean and simple common cores into reusable workflows. The mental mode is similar to a frontend/backend separation.

For your particular use case, we could run the matricized workflow using a bunch of single-element inputs, essentially launching a 1x1x1x...x1 matrix :grin: